packages/flutter/test/material/slider_theme_test.dart

```diff
@@ -548,4 +548,244 @@ void main() {
     );
     await gesture.up();
   });
+
```

Comment:
I haven't checked this whole file (so it may already exist) but if it doesn't, make sure you add "regression" test, that tests: when no custom track height/slider thumb shape/slider tick marks are provided, the defaults are the same as they were before this change.

Reply:
The tests above check for the default painting of shapes, so I think we are good there